waterlog/src/reducers/index.js

```diff
@@ -4,10 +4,12 @@ import SegmentLeaksReducer from './SegmentLeaksReducer';
 import SegmentLeaksByIdReducer from './SegmentLeaksByIdReducer';
 import LeakLitresReducer from './LeakLitresReducer';
 import LeaksResolvedReducer from './LeaksResolvedReducer';
+import SegmentEventsReducer from './SegmentEventsReducer';
 
 export default combineReducers({
 	leaks: SegmentLeaksReducer,
 	leak: SegmentLeaksByIdReducer,
 	litres: LeakLitresReducer,
-	leaksResolves: LeaksResolvedReducer 
+	leaksResolves: LeaksResolvedReducer,
+    events: SegmentEventsReducer
 });
```

Comment:
add new lines at the end of all files please

Reply:
Sure I fixed it.